functions/backup.sh

```diff
@@ -111,28 +107,36 @@ create_backup_config() {
 }
 
 amanda_setup() {
-
-  querytext="So you are about to install the Amanda backup solution.\nDocumentation is available at the previous openHABian menu point,\nat /opt/openhabian/docs/openhabian-amanda.md or at https://github.com/openhab/openhabian/blob/master/docs/openhabian-amanda.md\nHave you read this document ?"
-  introtext="This will setup a backup mechanism to allow for saving your openHAB setup and modifications to either USB attached or Amazon cloud storage.\nYou can add your own files/directories to be backed up, and you can store and create clones of your openHABian SD card to have an all-ready replacement in case of card failures."
-  failtext="Sadly there was a problem setting up the selected option. Please report this problem in the openHAB community forum or as a openHABian GitHub issue."
-  successtext="Setup was successful. Amanda backup tool is now taking backups at 01:00. For further readings, start at http://wiki.zmanda.com/index.php/User_documentation."
+## Function to setup Amanda backup system
+  local querytext="So you are about to install the Amanda backup solution.\nDocumentation is available at the previous openHABian menu point,\nat /opt/openhabian/docs/openhabian-amanda.md or at https://github.com/openhab/openhabian/blob/master/docs/openhabian-amanda.md\nHave you read this document ?"
+  local introtext="This will setup a backup mechanism to allow for saving your openHAB setup and modifications to either USB attached or Amazon cloud storage.\nYou can add your own files/directories to be backed up, and you can store and create clones of your openHABian SD card to have an all-ready replacement in case of card failures."
+  local failtext="Sadly there was a problem setting up the selected option. Please report this problem in the openHAB community forum or as a openHABian GitHub issue."
+  local successtext="Setup was successful. Amanda backup tool is now taking backups at 01:00. For further readings, start at http://wiki.zmanda.com/index.php/User_documentation."
 
   if [ -n "$INTERACTIVE" ]; then
     if ! (whiptail --title "Amanda backup installation" --yes-button "Yes" --no-button "No, I'll go read it" --defaultno --yesno "$querytext" 10 80) then return 0; fi
     whiptail --msgbox "$introtext" 25 132
   fi
+  
+  /usr/sbin/exim --version 2>/dev/null >/dev/null
+  if [ $? -ne 0 ]; then
+     if ! (whiptail --title "No exim mail transfer agent" --yes-button "Continue" --no-button "Back" --defaultno --yesno "Seems exim is not installed as a mail transfer agent.\nAmanda needs to be able to send emails. You should stop installing Amanda here and install exim from the openHABian menu (option 2C) first, then return here.\nOnly continue if you know there's a working mail transfer agent other than exim on your system.\nDo you want to continue with Amanda installation ?" 15 80) then echo "CANCELED"; return 0; fi
```

Comment:
Please call the EXIM script directly. eg. `exim_setup()` if you need it installed.
Suggestionally, you change the text to or similar:
```

whiptail --title "No exim mail transfer agent" --yes-button "Install EXIM4" --no-button "MTA already exist, ignore installation" --defaultyes --yesno ...
```